### DISCUSSION TITLE
AltSnap inside remote desktop session

### DISCUSSION THREAD

**Discussion by chrisdtaylor** [RamonUnch/AltDrag#497] [General]
From my initial experiments it doesn't seem like AltSnap can work inside a remote desktop session.  That is, it will resize the session, but I want it to operate internal to the session as if the session were a native/local desktop.  Any idea if this is somehow already possible?

**Comment by RamonUnch** [comment]
You can use AltSnap inside a remote desktop session:
1) You must run AltSnap on the target/server computer to which you are connecting.
2) You must blacklist `mstsc.exe` or `msrdc.exe` on the local computer from which you are connecting or disable AltSnap there.
